rpc_jobs/irr_role_test.yml

```diff
@@ -0,0 +1,102 @@
+- project:
+    name: 'RPC-IRR-Jobs'
+    # Note: branch is the branch for periodics to build
+    #       branches is the branch pattern to match for PR Jobs.
+    series:
+      - master:
+          branch: master
+          branches: "master"
+    image:
+      - xenial:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+      - trusty:
+          IMAGE: "Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)"
+    context:
+      - base
+    ztrigger:
+      - pr:
+          CRON: ""
+      - periodic:
+          branches: "do_not_build_on_pr"
+          NUM_TO_KEEP: 10
+    jobs:
+      - 'RPC-IRR_{series}-{image}-{context}-{ztrigger}'
+
+- job-template:
+    # DEFAULTS
+    STAGES: >
+      Allocate Resources,
+      Connect Slave,
+      Prepare Deployment,
+      Run IRR Tests,
+      Cleanup,
+      Destroy Slave
+    branch: master
+    NUM_TO_KEEP: 30
+
+    # TEMPLATE
+    name: 'RPC-IRR_{series}-{image}-{context}-{ztrigger}'
+    project-type: workflow
+    concurrent: true
+    properties:
+      - build-discarder:
+          num-to-keep: "{NUM_TO_KEEP}"
+      # IRR REPOS ARE ADDED HERE
+      - github:
+          url: https://github.com/rcbops/rpc-maas
+    parameters:
+      - rpc_gating_params
+      - single_use_slave_params:
+         IMAGE: "{IMAGE}"
+         FLAVOR: "{FLAVOR}"
+         REGION: "{REGION}"
+         IRR_CONTEXT: "{context}"
+         IRR_IMAGE: "{image}"
+         IRR_SERIES: "{series}"
+      - string:
+          name: STAGES
+          default: "{STAGES}"
+          description: |
+            Pipeline stages to run CSV. Note that this list does not influence execution order.
+            Options:
+              Allocate Resources
+              Connect Slave
+              Prepare Deployment
+              Run IRR Tests
+              Cleanup
+              Destroy Slave
+    triggers:
+      - github-pull-request:
+          org-list:
+            - rcbops
+          github-hooks: true
+          trigger-phrase: '.*recheck_cit_all.*|.*recheck_cit_{image}_{context}.*'
+          only-trigger-phrase: false
+          white-list-target-branches:
+            - "{branches}"
+          auth-id: "github_account_rpc_jenkins_svc"
+          status-context: 'CIT/{image}_{context}'
+          cancel-builds-on-update: true
+
+    dsl: |
+      // CIT Slave node
+      timeout(time: 2, unit: 'HOURS'){{
+        node() {{
+          dir("rpc-gating") {{
+              git branch: env.RPC_GATING_BRANCH, url: env.RPC_GATING_REPO
+              common = load 'pipeline_steps/common.groovy'
+              pubcloud = load 'pipeline_steps/pubcloud.groovy'
+              run_tests = load 'pipeline_steps/run_tests.groovy'
+          }}
+          // try within pubcloud node so we can archive archive_artifacts
```

Comment:
This comment is misleading as the try it is after does not run on a pubcloud node. The whole try block could be removed as it doesn't do anything useful on catch or finally.